src/components/AgmLocationChange/AgmLocation.vue

```diff
@@ -52,6 +52,7 @@ export default class AgmLocation extends Vue {
   /** Emits an event with the changed input (ie, updated v-model). */
   @Emit('update:agmLocation')
   emitInput (val: string): void {
+    this.agmLocation = val
```

Comment:
I noticed that the agmLocation variable here wasn't being changed. This was causing the function on line 45 to ALWAYS trigger. Hence, the fields were always showing the they're not valid although they are just as Yui mentioned:

![yui comment agm location](https://github.com/bcgov/business-filings-ui/assets/122301442/c2f72cc0-03cb-4b96-9dc3-21383de15561)

After my update:
![after my update yui](https://github.com/bcgov/business-filings-ui/assets/122301442/1429095b-ebf7-42d6-871c-15573655192f)

It's the exact same thing for the other components.